Realm/RLMDecimal128.h

```diff
@@ -67,6 +73,49 @@ NS_ASSUME_NONNULL_BEGIN
 
 /// Gets if this Decimal128 represents a NaN value.
 @property (nonatomic, readonly) BOOL isNaN;
+
+/// The magnitude of this RLMDecimal128.
+@property (nonatomic, readonly) RLMDecimal128 *magnitude NS_REFINED_FOR_SWIFT;
+
+/// Replaces this RLMDecimal128 value with its additive inverse.
+- (void)negate NS_REFINED_FOR_SWIFT;
```

Comment:
Seems to me that `negate` is the only one here that should _not_ be `NS_REFINED_FOR_SWIFT` while the rest should be.

Reply:
I see now that you were just mirroring the implementation of other swift Number structs, but I still find this a bit odd.